reductus/reflred/angles.py

```diff
@@ -37,8 +37,11 @@ def apply_divergence_simple(data, sample_width):
     dtheta = divergence_simple(slits=slits, distance=distance, T=theta,
                                sample_width=sample_width, use_sample=use_sample)
     #print("divergence", theta.shape, dtheta.shape)
-    if data.source_divergence is not None:
-        dtheta = np.minimum(dtheta, FWHM2sigma(data.source_divergence/2))
+    if data.source_divergence_fw is not None:
+        # source_divergence_fw represents a uniform distribution:
+        # full-width = sqrt(12) * sigma
+        source_divergence_sigma = data.source_divergence_fw / np.sqrt(12.0)
```

Comment:
Note that if `source_divergence_fw` corresponds to a uniform distribution, then the conversion to a 1-sigma width is $\sigma = \frac{FWHM} {\sqrt{12}}$ instead of the conversion `FWHM2sigma` we were using...

@acaruana2009 can you check if this affects your results?